xcode/Subconscious/Shared/Services/OnThisDayGeist.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  OnThisDayGeist.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 30/9/2022.
+//
+
+import Foundation
+
+
+enum OnThisDayVariant: CaseIterable {
+    case inTheLastDay
+    case inTheLastWeek
+    case aYearAgo
+    case sixMonthsAgo
+    case aMonthAgo
+    
+    static func random() -> OnThisDayVariant {
+        // https://stackoverflow.com/questions/63189870/return-a-random-case-from-an-enum-in-swift
+        return allCases.randomElement()!
+    }
+}
+
+struct OnThisDayGeist: Geist {
```

Comment:
This _might_ be better named `MementoGeist` or something similar. `OnThisDay` is clear but the `inTheLastDay` and `inTheLastWeek` cases feel discordant with the name. 

_Maybe_ I'm overthinking it.

Reply:
Memento
Recollect
Remember

I like this idea. Choose your fav!